src/main/java/org/folio/circulation/services/support/ItemsInTransitReportContext.java

```diff
@@ -2,6 +2,9 @@
 
 import java.util.Map;
 
+import lombok.AllArgsConstructor;
```

Comment:
Please use all Lombok imports in one place

Reply:
done